__tests__/app.test.js

```diff
@@ -111,3 +112,49 @@ describe("/api/articles", () => {
   })
   
 });
+describe("/api/articles/:article_id/comments", () => {
+  test("get all comments for a specified article", () => {
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then(({body}) => {
+      const {comments} = body;
+      expect(comments).toHaveLength(11);
+      comments.forEach(comment => {
+      expect(comment).toMatchObject({
+        comment_id: expect.any(Number),
+        body: expect.any(String),
+        article_id: expect.any(Number),
+        author: expect.any(String),
+        votes: expect.any(Number),
+        created_at: expect.any(String)
+      })  
+      })
+      });
+  });
+
+  test("GET:404 sends an appropriate status and error message when article id given is valid but non-existen id", () => {
+        return request(app)
+        .get("/api/articles/99/comments")
+        .expect(404)
+        .then((response) => {
+        expect(response.body.msg).toBe("this article doesn't have a comment");
+      });
+})
+  test("GET:400 sends an appropriate status and error message when given an invalid id", () => {
+    return request(app)
+    .get("/api/articles/banana/comments")
+    .expect(400)
+    .then((response) => {
+      expect(response.body.msg).toBe("Bad request");
+    });
+})
+  test("GET:404 sends an appropriate status and error message when given valid ID but the article has no comments", () => {
+    return request(app)
+      .get("/api/articles/7/comments")
+      .expect(404)
```

Comment:
This should be a 200

Reply:
The article exists but the article has no comments.  it is a valid response to get.